### DISCUSSION TITLE
Looking to get started with working on open source projects.

### DISCUSSION THREAD

**Discussion by AndreyKarmanov** [extesy/hoverzoom#719] [Q&A]
Hey! I am fairly young and trying to find a small project that I can help out, getting me some experience with GitHub and whatnot. 
I saw that there was a TODO in the /common.js with migrating the storage to google's storage system, and was going to make a branch to work on that (since I've done something similar before and seems fairly simple). Are there any plans/requirements for the migration?

If there is something smaller that I can work on that someone could allocate me that would be cool too.

**Comment by extesy** [comment]
Hi Andrey, there is a long list of potential issues and improvements on https://github.com/extesy/hoverzoom/issues page - feel free to pick something and send a pull request. But please make sure that the code you write is not Chrome-specific because the same extension is also distributed to Firefox and Edge extension stores.

**Reply by AndreyKarmanov** [reply]
Great, Thanks!
